### PR TITLE
feat: Add GraphicsContextSettings

### PR DESCRIPTION
**Pull request by kgoedecke**
*Description of changes:* Adds the GraphicsContext class.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by kgoedecke** [comment]
@dbanksdesign its being used for example to set the Opacity:

```
"contextSettings": {
  "_class": "graphicsContextSettings",
  "blendMode": 0,
  "opacity": 0.4974423363095238
},
```

**Comment by L2jLiga** [comment]
Would you like to change Shadow/Style models? It seems like they're using graphicsContextSettings.

https://github.com/amzn/sketch-constructor/blob/master/models/Shadow/index.d.ts#L8-L12

https://github.com/amzn/sketch-constructor/blob/master/models/Style/index.d.ts#L53-L57

**Comment by dbanksdesign** [comment]
Good catch @L2jLiga ! @kgoedecke can you update the Shadow and Style models to reference the new GraphicsContextSettings model? Thanks!

**Comment by kgoedecke** [comment]
@L2jLiga @dbanksdesign I added the changes for Shadow and Style.

File: models/GraphicsContextSettings/GraphicsContextSettings.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    }
+    return this;
+  }
+}
+
+/**
+ * @property {integer} blendMode
+ * @property {float} opacity
+ */
+GraphicsContextSettings.Model = {
```
Comment (L2jLiga) [inline models/GraphicsContextSettings/GraphicsContextSettings.js]: Would like to mention, following discussion in #45 this should be static getter

Comment (kgoedecke) [reply models/GraphicsContextSettings/GraphicsContextSettings.js]: Yeah. I would almost say you merge your PR first and I change my PR.

Comment (L2jLiga) [reply models/GraphicsContextSettings/GraphicsContextSettings.js]: We can do it simultaneously 😉

Comment (kgoedecke) [reply models/GraphicsContextSettings/GraphicsContextSettings.js]: Cherry Pick?

Comment (L2jLiga) [reply models/GraphicsContextSettings/GraphicsContextSettings.js]: no, I mean just change your model

Comment (kgoedecke) [reply models/GraphicsContextSettings/GraphicsContextSettings.js]: @L2jLiga updated according to https://github.com/amzn/sketch-constructor/pull/52

**Comment by kgoedecke** [comment]
@dbanksdesign should be good for a final 👀

**Comment by dbanksdesign** [comment]
Looks good! just need to resolve the 2 conflicts which should be easy (from merging @L2jLiga getter PR).

**Comment by kgoedecke** [comment]
@dbanksdesign done :)

**Pull request merged**